app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord  
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  devise :database_authenticatable, :confirmable , :registerable, :recoverable, :rememberable, :validatable
+  MIN_PASSWORD_LENGTH = 7
+  validates_acceptance_of :terms, :allow_nil => false, :accept => true
+end
```

Comment:
add a new line between validations, constants and devise code as these 3 things are separate things